pkg/plugins/platform/utils.go

```diff
@@ -35,10 +37,22 @@ func GetApps() map[string][]App {
 				if err != nil {
 					log.Fatalf("could not get application for plugin type %s", platformType)
 				}
+
 				approot, ok := siteContainer.Labels["com.ddev.approot"]
 				if !ok {
 					break
 				}
+				// Check to see if the app's root directory exists.
+				exists, _ := exists(approot)
+				if !exists {
+					// Create a temporary config file to store info about the deleted site.
+					tempConfig, _ := ddevapp.NewConfig(approot, "")
+					tempConfig.Name = siteContainer.Labels["com.ddev.site-name"]
+					tempSite := &LocalApp{tempConfig}
+					apps[platformType] = append(apps[platformType], tempSite)
+					break
```

Comment:
This break feels kind of weird? what are we breaking from?

Reply:
This break stops the `site` variable from running `.Init(<app_root>)` a few lines below